src/globals.ts

```diff
@@ -490,6 +490,7 @@ export const SupportedLocales = Object.freeze([
   { name: 'Português', code: 'pt' },
   { name: 'Русский', code: 'ru' },
   { name: 'Slovenščina', code: 'sl' },
+  { name: 'Türkçe', code: 'tr' },
```

Comment:
Hi @fatih5228, can you confirm this is the correct translation for "Turkish" language?

Reply:
Hi. @pedrolamas, the translation is correct.  Thank you